scipy/stats/tests/test_distributions.py

```diff
@@ -2879,6 +2886,7 @@ def test_issue_12796(self):
         res = stats.beta.cdf(inv, a, b)
         assert_allclose(res, 1 - alpha_2)
 
+    @pytest.mark.skipif(MACOS_INTEL, reason="Overflow, see gh-14901")
```

Comment:
I thought the third failing test was `test_issue_12635` above, unless I'm misreading the diff

```
 FAILED scipy/stats/tests/test_distributions.py::TestBeta::test_issue_12635 - ...
FAILED scipy/stats/tests/test_distributions.py::TestBeta::test_issue_12794 - ...
FAILED scipy/stats/tests/test_distributions.py::TestBeta::test_issue_12796 - ...
```

Reply:
I went ahead and pushed that in--probably just some speedy copy-paste work.